mock-recipe-server/bin/ci/deploy

```diff
@@ -1,4 +1,11 @@
 #!/usr/bin/env bash
-set -eu
+set -e
+
+if [[ -z $MOCK_SERVER_ARTIFACTS || -z $MOCK_SERVER_S3_BUCKET ]]; then
```

Comment:
Ah, yeah. [`AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`](http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html#cli-environment).